docs/sdks/kotlin-sdk/_category_.json

```diff
@@ -0,0 +1,5 @@
+{
+  "label": "Kotlin SDK",
+  "position": 1,
```

Comment:
I think I uses position 1 for swift. Not sure if it matters though since it seems to render ok.

Reply:
You stole position 1?! 😮 I'm not sure it matters though since SDKs is auto-generated. This might override it, but if two have the same, it probably defaults to the auto-generated.